### ISSUE TITLE
vcs: line endings after clone

### ISSUE THREAD

**Issue by Carbenium** [arfbtwn/banshee#56]
Since b2fcbae8fb1fe857c36fdb6545532dbc7d78b79d git isn't happy with the line endings after a fresh clone.

```
> git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   help/es/es.po
	modified:   tests/data/amz/decrypted.amz
	modified:   tests/data/xspf/xspf-1_0.2.xsd

no changes added to commit (use "git add" and/or "git commit -a")
```
`core.autocrlf` is set to `input`.

**Comment by arfbtwn** [comment]
couple of the line ending settings in that commit may be wrong but I just tested with `core.autocrlf` set to `input` globally and it didn't occur on my system when I cloned. What's the output from `git config -l`?

**Comment by Carbenium** [comment]
Here you go: 
```core.autocrlf=input
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
```

**Comment by arfbtwn** [comment]
Odd, I retried with `core.filemode` set to `true` as I had that `false` but I still couldn't reproduce. What does `git diff` say about the difference, is it definitely line endings?

**Comment by Carbenium** [comment]
Well, just solved it by upgrading `git` from 2.7.4 to 2.14.2.

**Comment by arfbtwn** [comment]
Cool, mind if we close this one?
